docs/Manifest.toml

```diff
@@ -70,9 +70,9 @@ version = "0.1.0"
 
 [[CLIMAParameters]]
 deps = ["Test"]
-git-tree-sha1 = "9c3eae7d6bcabf604d9e4e9cc0802db01357bc37"
+git-tree-sha1 = "540e1206343d84106bc989054d771ad1e0eaaa14"
 uuid = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
Shouldn't the docs use the same version of CLIMAParameters.jl as ClimateMachine.jl?

Reply:
yeah - Thats exactly what I'm trying to fix now. But for some reason they just do not want to update CLIMAParameters

Reply:
I managed to up the CLIMAParameters but the KernelAbstractions  are stuck at KernelAbstractions v0.2.3. @charleskawczynski - do you know what may be blocking them?

Reply:
I'm confused now. After I upgraded  to `CLIMAParameters v1.6` in the docs I started getting errors from the docs build. I think the reason why is that the docs themselves depend on `ClimateMachine v0.1.0` and which is not compatible with `CLIMAParameters v1.6`. 

How do I get out of this dependency hell? Help...

Reply:
I think I got out!